_data/en/settings.yml

```diff
@@ -59,7 +59,7 @@ contact_page:
       'FDIC Resolution Request Management Portal': FDIC Resolution Request Management Portal
       'FEMA Disaster Assistance': FEMA Disaster Assistance
       'General Services Administration': General Services Administration
-      'Get .gov': Get .gov
+      'Get .gov': Get.gov
```

Comment:
do we want the space still for the regular value. otherwise this is good

Reply:
The id value has not been changed in Salesforce yet. The SF team wanted the visual change to be made immediately and then their team will update the value in their backend by May 1.

Reply:
Created a ticket for the `id` once SF makes their change: [LG-12812: Change "Get .gov backend value](https://cm-jira.usa.gov/browse/LG-12812)